### ISSUE TITLE
Missing native CSS function: minmax()

### ISSUE THREAD

**Issue by idoros** [wix/stylable#550]
A warning for `cannot find native function or custom formatter called minmax` is shown when using [minmax()](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax) value:

```css
.root {
    display: grid;
    grid-template-columns: minmax(150px, min-content) 3fr;
}
```

**Comment by tomrav** [comment]
Good catch, will be added.

**Comment by tomrav** [comment]
Fixed in `@stylable/core` version `1.0.6`.
